src/content/docs/style-guide/writing-guidelines/pricing-language-guidelines.mdx

```diff
@@ -9,35 +9,38 @@ This doc explains required styles and recommended phrasings for pricing-related
+* We have a high level of pricing/billing detail in the [pricing/billing-related docs](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/). 
+* When a feature requires a pricing edition (Standard, Pro, Enterprise), we should place that in the requirements of a feature's doc. Because there are only a few things in the Pro and Enterprise editions, this should be an infrequent need.
 
-Here are some reasons behind this philosophy of minimizing pricing-related docs mentions:
+## Style and formatting [#style-format]
 
-### New Relic One pricing plan [#nr-one-plan]
+Some notes on style and formatting for pricing-related language: 
 
-For the New Relic One pricing plan, there are several pricing factors involved. As a general approach, the only pricing factor we document is that related to the editions (Standard, Pro, and Enterprise).
+* Pricing plans: 
+    * Use the word "plan" to refer **only** to our two pricing plans: New Relic One pricing plan, and our original pricing plan. Our pricing editions (Standard, Pro, Enterprise) are not "plans"; they are "editions". Our "free tier" is also not a "plan."
+    * Examples of referring to plans: 
+        * *If you're on our New Relic One pricing plan...*
+        * *If you're on our original pricing plan...*
+    * When referencing a plan, it's good practice to point to either the doc for that plan, or else the [doc explaining the differences between the plans](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model/#pricing-user-table), whichever makes more sense. 
+* "usage-based." The New Relic One pricing plan can be referred to as "usage-based," but not "consumption-based."
+* Editions:  
+    * Our pricing editions are formatted with title case: Standard, Pro, and Enterprise. Example use: *If your organization is on the [Pro edition](https://newrelic.com/pricing)...*
+    * For recommendations for including the edition in feature requirements, see [edition guidelines](#edition-guidelines). 
+* Free tier. The so-called New Relic One pricing plan's [free tier](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#free) isn't actually a specific plan or a specific edition. There's nothing special about it. It is simply a situation where an organization is on the Standard edition but isn't yet paying money (either because they haven't put in a credit card or because they're under the free limits of usage). 
+* Usage plans: 
+    * The New Relic One pricing plan has two usage plans: pay-as-you-go and annual pool of funds. 
+    * Example use: *On the pay-as-you-go usage plan, your organization...*
+    * Do not use abbreviations for these usage plans in public-facing communication.  
+* Billable users. For how to reference basic users and full users, see [User-related language guidelines](/docs/style-guide/writing-guidelines/user-related-language-guidelines). 
 
```

Comment:
This all looks good content wise, but unfortunately it looks like something broke your build so I can't preview it without all the .mdx clutter...want to tag me when you get the preview issue worked out?